protocol/communities/manager.go

```diff
@@ -726,8 +726,10 @@ func (m *Manager) LeaveCommunity(id types.HexBytes) (*Community, error) {
 	if community == nil {
 		return nil, ErrOrgNotFound
 	}
+  community.RemoveUserFromOrg(m.identity)
```

Comment:
Notice that I'm not handling possible errors here. When this API is called we know that the user exists in the org so removal should always go fine.

Reply:
nit: github shows some indentation issue

Reply:
Fixed!